manifests/0000_30_cluster-api_12_clusteroperator.yaml

```diff
@@ -31,3 +31,110 @@ status:
     name: cluster-capi-operator
     namespace: openshift-cluster-api
     resource: deployments
+  - group: "cluster.x-k8s.io"
+    name: ""
+    namespace: openshift-cluster-api
+    resource: clusters
+  - group: "cluster.x-k8s.io"
+    name: ""
+    namespace: openshift-cluster-api
+    resource: machinesets
+  - group: "cluster.x-k8s.io"
+    name: ""
+    namespace: openshift-cluster-api
+    resource: machines
+  - group: "cluster.x-k8s.io"
+    name: ""
+    namespace: openshift-cluster-api
+    resource: machinedeployments
+
+  - group: "infrastructure.cluster.x-k8s.io"
+    name: ""
+    namespace: openshift-cluster-api
+    resource: awsclusters
```

Comment:
If you're actively managing these entries in your operator (which it looks like you are), I don't think you need to go through the work of manually listing them here in your ClusterOperator manifest.  You need enough in the manifest that a must-gather while your operator isn't running collects enough to figure out why the operator isn't running.  And it seems unlikely that folks would need to get all the way out to cloud-specific types to figure out why the operator wasn't running?

But also, 🤷, if you don't mind managing this list by hand or you find some way to automate it, having the cloud-specific types in this manifest doesn't hurt.